src/vcpkg-test/system.process.cpp

```diff
@@ -41,6 +41,27 @@ TEST_CASE ("captures-output", "[system.process]")
     REQUIRE(run.output == expected);
 }
 
+#ifdef _WIN32
```

Comment:
Does this need to be guarded for _WIN32? It seems like it should pass on other platforms...